src/main/java/com/project/stockexchangeappbackend/service/StockServiceImpl.java

```diff
@@ -72,14 +77,17 @@ public Stock getStockByAbbreviation(String abbreviation) {
     @LogicBusinessMeasureTime
     @Transactional
     public Stock updateStock(Stock stock) {
-        return stockRepository.save(stock);
+        Stock updatedStock = stockRepository.save(stock);
+        log.info("Stock with id " + updatedStock.getId() + " was successfully updated");
+        return updatedStock;
     }
 
     @Override
     @LogicBusinessMeasureTime
     @Transactional
     public void updateStocks(Collection<Stock> stocks) {
-        stockRepository.saveAll(stocks);
+        List<Stock> updatedStocks = stockRepository.saveAll(stocks);
+        updatedStocks.forEach(stock -> log.info("Stock with id " + stock.getId() + " was successfully updated"));
```

Comment:
You can omit saving reference to var.You don't return anything, so you can do as follow: stockRepository.saveAll(param).forEach()